<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageItem</class>
 <widget class="QDialog" name="ManageItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManageItem</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="ItemGroupBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>311</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Item</string>
   </property>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Item Code:</string>
    </property>
   </widget>
   <widget class="QLabel" name="itemCode">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="invoiceNo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Invoice No.:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Item Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="collctedDateLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Collected Date:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="statusComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Waiting</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ready</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Collected</string>
     </property>
    </item>
   </widget>
   <widget class="QDateTimeEdit" name="collecteddateTimeEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Invoice No./ Item Code:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="invoiceNoItemCode">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="itemTableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="loadComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Please select status</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Waiting</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ready</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Collected</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>All</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
